### DISCUSSION TITLE
Unclear benefit of using jest executor

### DISCUSSION THREAD

**Discussion by GeorgeTaveras1231** [nrwl/nx#14042] [General]
Hello 

I'm currently working on setting up jest in my nx project. I've added a `project.json` to each workspace that has jest tests with the jest executor as such

```json
{
  "targets": {
    "test": {
      "executor": "@nrwl/jest:jest",
      "options": {
        "jestConfig": "./lib-ui/packages/dummy-component/jest.config.mjs"
      }
    }
  }
}
```

However, this option seems a lot more verbose and with more boilerplate than simply creating a `test` script in the `package.json` file as such

```json
{
  "scripts": {
    "test": "jest"
  }
}
```

And from following the documentation [here](https://nx.dev/packages/jest/executors/jest) it is not clear wether I'd lose some benefit by simply doing the latter which seems more appealing because of the reduced boilerplate. 

Can someone explain, is there a benefit to doing the former -- explicitly using the nx executor vs simply using the jest CLi in each workspace?

---
> Note: I've also read the docs on executors [here](https://nx.dev/plugin-features/use-task-executors#use-task-executors) and am still not totally sold, and think I'm missing misunderstanding or missing something 🤔

**Comment by rambochase** [comment]
您好，我是黎锐标，您的来信我已收到！

**Comment by wujinkui** [comment]
您好，我已经收到您的邮件。
